wordpress-mobile/WordPress-Android
# Issue/8647 add new qs tutorials

Fixes #8647

This PR adds new Quick Start tutorials:

**Upload a site icon**
[![Image from Gyazo](https://i.gyazo.com/b1796ccc534e950ea8b161b5a8b655e3.gif)](https://gyazo.com/b1796ccc534e950ea8b161b5a8b655e3)

**Create a new page**
[![Image from Gyazo](https://i.gyazo.com/31e58bd0857f9e2da7bd4f58cd31405b.gif)](https://gyazo.com/31e58bd0857f9e2da7bd4f58cd31405b)

**Check your site stats**
[![Image from Gyazo](https://i.gyazo.com/3e99517d1a9b3eee420202becc8b4348.gif)](https://gyazo.com/3e99517d1a9b3eee420202becc8b4348)

**Explore plans**
[![Image from Gyazo](https://i.gyazo.com/8594ffa9dc5b87cdeaab8db5b491e77e.gif)](https://gyazo.com/8594ffa9dc5b87cdeaab8db5b491e77e)

In addition, this PR removes Quick Start notices, so the users no longer be prompted to do/continue specific tasks.

To test:
Try new tutorials and make sure they work, and quick start notices do not appear in the process.
